WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -883,7 +883,7 @@ extension WordPressAppDelegate {
         let maximumPointSize = WPStyleGuide.maxFontSize
 
         UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white,
-                                                            NSAttributedString.Key.font: WPStyleGuide.fixedFont(for: UIFont.TextStyle.headline, weight: UIFont.Weight.bold)]
+                                                            NSAttributedString.Key.font: WPStyleGuide.fixedFont(for: UIFont.TextStyle.headline, weight: UIFont.Weight.semibold)]
```

Comment:
For more info on this change check the commit message: https://github.com/wordpress-mobile/WordPress-iOS/pull/14367/commits/10776b6576e878d8c70eaf36615c141fa1127053